index.html

```diff
@@ -34,12 +34,163 @@ <h1>I’m Omayio Glad to see you!</h1>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
       </section>
+      <section id="about-me" class="flex">
+        <h2>About Myself</h2>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h5>LET’S CONNECT</h5>
+        <ul class="flex">
+          <li><a href="#"><img src="imgs/Github.png" alt="Github Icon" /></a></li>
+          <li><a href="#"><img src="imgs/Linkedin-icon.png" alt="Linkedin-icon" /></a></li>
+          <li><a href="#"><img src="imgs/hands.png" alt="hands" /></a></li>
+          <li><a href="#"><img src="imgs/Twitter.png" alt="Twitter Icon" /></a></li>
+          <li><a href="#"><img src="imgs/M.png" alt="M icon" /></a></li>
+        </ul><br><br>
+        <button type="submit">Get My Resume</button>
+        <ul class="lang-section">
+          <li>
+            Languages
+            <img src="imgs/Icon-Down.png" alt="#" />
+          </li>
+        </ul>
+        <div class="lang-prog">
+          <img class="javascript-img" src="imgs/JS.png" alt="Javascript">
+          <span> &nbsp; &nbsp; JavaScript</span>
+        </div>
+
+        <div class="lang-prog">
+          <img class="html-img" src="imgs/HTML.png" alt="HTML">
+          <span> &nbsp; &nbsp; HTML</span>
+        </div>
+
+        <div class="lang-prog">
+          <img class="css-img" src="imgs/CSS.png" alt="css">
+          <span> &nbsp; &nbsp; JavaScript</span>
+        </div>
+
+
+        <ul class="lang-section hg">
+          <li>
+            Frameworks
+            <img src="imgs/Icon-Right.png" alt="" />
+          </li>
+          <li>
+            Skills <img src="imgs/Icon-Right.png" alt="" />
+          </li>
+        </ul>
+      </section>
+
+    </div>
+
+    <footer>
+      <h3>Contact me</h3>
+      <p>
+        If you have an application you are interested in developing, 
+        a feature that you need built or a project that needs coding. 
+        I’d love to help with it 
+      </p>
+      <form action="https://formspree.io/f/2" method="post">
+          <input class="form" type="text" maxlength="30" placeholder="Your name" required> <br>
+          <input class="form" type="email" placeholder="Enter Email" required> <br>
+          <textarea class="form" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>       
+          <input class="form-btn" type="button" value="Get it Touch"> <br>
```

Comment:
### Kindly fix the button's style 👍🏻 

#### Your Button's style
<img width="144" alt="Screen Shot 2022-04-28 at 4 33 43 AM" src="https://user-images.githubusercontent.com/83048208/165647979-23824040-1a24-4660-8fc9-0a99b75e20df.png">

#### Required button's style
<img width="135" alt="Screen Shot 2022-04-28 at 4 34 51 AM" src="https://user-images.githubusercontent.com/83048208/165648063-f31fd73e-a6ba-4116-b88d-8b1c6a662809.png">

Reply:
@zainsadaqat , This is now corrected to the required button style